### ISSUE TITLE
Support PHP 7.4

### ISSUE THREAD

**Issue by MarvinKlar** [instaclick/php-webdriver#107]
Hello everyone,

I was about to update a project to PHP 7.4. While doing so I got the following error:
```
Behat\Testwork\Call\Exception\CallErrorException: Notice: Trying to access array offset on value of type null in vendor/instaclick/php-webdriver/lib/WebDriver/AbstractWebDriver.php line 156 in vendor/behat/behat/src/Behat/Testwork/Call/Handler/RuntimeCallHandler.php:90
      Stack trace:
      #0 vendor/instaclick/php-webdriver/lib/WebDriver/AbstractWebDriver.php(156): Behat\Testwork\Call\Handler\RuntimeCallHandler->handleError()
      #1 vendor/instaclick/php-webdriver/lib/WebDriver/WebDriver.php(70): WebDriver\AbstractWebDriver->curl()
      #2 vendor/behat/mink-selenium2-driver/src/Selenium2Driver.php(300): WebDriver\WebDriver->session()
...
```

I only get this error on PHP 7.4. Am I missing something or is this a bug? Would you mind fixing it if so?

**Comment by MarvinKlar** [comment]
This should be [this line](https://github.com/instaclick/php-webdriver/blob/master/lib/WebDriver/AbstractWebDriver.php#L228): `if ((int) $result['status'] !== 0) {`

**Comment by GautierDig** [comment]
Same issue here. Any update on this ?

**Comment by robocoder** [comment]
Which driver binary (and version) are you all using?

**Comment by GautierDig** [comment]
I am using the chromedriver 91.0.4472.101

**Comment by robocoder** [comment]
Please also indicate which PHP version you're running. A couple of (major?) curl POSTFIELDS bugs were fixed in 7.4.2 and 7.4.15.

**Comment by GautierDig** [comment]
I am using PHP 7.4.15

**Comment by robocoder** [comment]
Please confirm this is fixed for you in 1.4.8. Thanks.

**Comment by GautierDig** [comment]
I updated to 1.4.9, I got an error:
`WebDriver\Exception\CurlExec: Failed to connect to localhost port 4444: Connection refused in vendor/instaclick/php-webdriver/lib/WebDriver/Exception.php:155` 
I downgrade to 1.4.8 and it worked as expected.
